be/src/olap/version_graph.cpp

```diff
@@ -57,7 +57,7 @@ void TimestampedVersionTracker::construct_versioned_tracker(
     _next_path_id = 1;
     _construct_versioned_tracker(rs_metas);
 
-    // init _stale_version_path_map
+    // Init '_stale_version_path_map'.
```

Comment:
```suggestion
    // Init `_stale_version_path_map`.
```

Use  ` would be better. 

We can see `Function Comments` in `google cpp style`. [here](https://google.github.io/styleguide/cppguide.html#Comment_Style)